ple/games/donkeykong/Board.py

```diff
@@ -0,0 +1,338 @@
+__author__ = 'Batchu Vishal'
+import pygame
+import math
+import random
```

Comment:
please use numpy for the random functions.

can replace this with `import numpy as np`